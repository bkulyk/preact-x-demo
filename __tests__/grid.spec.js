import React from 'react';
import Grid from '@bitchin/grid';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('should render correctly', () => {
  expect(shallow(<Grid>Something</Grid>)).toMatchSnapshot();
});

it('should render correctly with props', () => {
  expect(shallow((
    <Grid
      data-something="test"
      className="custom"
      align="right"
      fixedColumnWidth
    >
      Something
    </Grid>
  ))).toMatchSnapshot();
});
