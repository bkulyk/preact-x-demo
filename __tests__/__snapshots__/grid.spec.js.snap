// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="mdc-layout-grid"
>
  Something
</div>
`;

exports[`should render correctly with props 1`] = `
<div
  className="custom mdc-layout-grid mdc-layout-grid__cell--align-right mdc-layout-grid--fixed-column-width"
  data-something="test"
>
  Something
</div>
`;
